RiotSharp.Test/RiotApiTest.cs

```diff
@@ -133,12 +133,12 @@ public void GetChampions_FreeToPlay_Test()
                 var champions = api.GetChampions(RiotApiTestBase.summoner1and2Region, true);
```

Comment:
Can you add the [EnsureCredibility method](https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp.Test/RiotApiTest.cs#L25) here and remove the `Ignore` attribute.